### PR TITLE
Create a common Abstract Base class with common fields

### LINKED ISSUE #6: Create an abstract model with common fields in the common app
An abstract class to hold all the common fields

### PR DESCRIPTION
**Pull request by yoda-yoda**
closes #6

### CODE REVIEW

File: kulobapp/common/models/base.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from django.utils import timezone
+
+
+class AbstractBase(models.Model):
+    """An Abstract class to hold major required fields."""
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    created = models.DateTimeField(default=timezone.now)
+    updated = models.DateTimeField(default=timezone.now)
+    views = models.IntegerField(default=0)
```
Comment (ikosenn) [inline kulobapp/common/models/base.py]: views in terms of? Page views...

File: kulobapp/common/models/base.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    active = models.BooleanField(default=True)
+    deleted = models.BooleanField(default=False)
+
+    def delete(self):
+        """Override DRF delete to do fake deletes."""
+        self.deleted = True
+        self.save()
+
+    class Meta(object):
```
Comment (ikosenn) [inline kulobapp/common/models/base.py]: can we make default ordering here to be created?

File: kulobapp/common/models/base.py (hunk 1 above)
Comment (yoda-yoda) [inline kulobapp/common/models/base.py]: Yeah yeah page views, i was thinking we could have that counter to get to know stuff people are more interested in before we figure out a more "intelligent" way

File: kulobapp/common/models/base.py (hunk 2 above)
Comment (yoda-yoda) [inline kulobapp/common/models/base.py]: Like so? What if we did it per view?

**Pull request merged**